decidim-core/spec/system/notifications_spec.rb

```diff
@@ -111,4 +111,20 @@
       end
     end
   end
+
+  context "with user group mentioned notifications" do
```

Comment:
I'm not a fan of having the system spec for this, but at least it's failing before this fix.

Ideally we could reproduce this bug and catch it in the event's spec (`decidim-comments/spec/events/decidim/comments/user_group_mentioned_event_spec.rb`) but that seems to work right, so I'm OK with the system spec approach. :+1: